mne/io/reference.py

```diff
@@ -97,11 +128,16 @@ def _apply_reference(inst, ref_from, ref_to=None):
                 'using the apply_proj() method function before the desired '
                 'reference can be set.'
             )
+
     for i in projs_to_remove:
         del inst.info['projs'][i]
```

Comment:
It is common to have the EEG channels in avg ref and EOG in bipolar. To the best of my knowledge, having some EEG in bipolar and some EEG in avg ref is rare. If we can support it in a clean way though, we should go for it. If the avg ref is not applied yet, we could update it. If it has been applied, then just leave it be.